numpy/core/tests/test_records.py

```diff
@@ -132,7 +132,9 @@ def test_0d_recarray_repr(self):
                       dtype=[('f0', '<i4'), ('f1', '<f8'), ('f2', '<M8[Y]')])"""))
 
         record = arr_0d[()]
-        assert_equal(repr(record), "(1, 2., '2003')")
+        assert_equal(repr(record), 
+            "np.record((1, 2.0, '2003'), "
+            "dtype=[('f0', '<i4'), ('f1', '<f8'), ('f2', '<M8[Y]')])")
         # 1.13 converted to python scalars before the repr
         try:
```

Comment:
Another place where it could be `with np.printoptions(...)`

Reply:
There is a lot of this, not going to touch it here.  Also, I would hope we can get rid of 1.13?

Reply:
OK, makes sense.

For 2.0, yes, seems to make sense to get rid of 1.13.

Reply:
(though not in this PR!)